package.json

```diff
@@ -15,19 +15,19 @@
     "build": "kolibri-tools build prod --file ./build_tools/build_plugins.txt && yarn run hashi-build",
     "makemessages": "kolibri-tools build i18n --file ./build_tools/build_plugins.txt",
     "watch": "kolibri-tools build dev --file ./build_tools/build_plugins.txt",
-    "watch-hot": "kolibri-tools build dev --file ./build_tools/build_plugins.txt --hot",
+    "watch-hot": "yarn run watch --hot",
     "django-devserver": "kolibri --debug manage runserver --settings=kolibri.deployment.default.settings.dev \"0.0.0.0:8000\" ",
-    "devserver": "npm-run-all --parallel django-devserver watch lint:watch hashi-dev",
-    "devserver-hot": "npm-run-all --parallel django-devserver watch-hot lint:watch",
+    "devserver": "npm-run-all --parallel django-devserver fmt-frontend:watch fmt-backend:watch hashi-dev watch",
+    "devserver-hot": "npm-run-all --parallel django-devserver fmt-frontend:watch fmt-backend:watch hashi-dev watch-hot",
     "bundle-stats": "kolibri-tools build stats --file ./build_tools/build_plugins.txt",
     "clean": "kolibri-tools build clean --file ./build_tools/build_plugins.txt",
     "preinstall": "node ./packages/kolibri-tools/lib/npm_deprecation_warning.js",
-    "lint": "kolibri-tools lint",
-    "lint-all": "yarn run lint '{kolibri*/**/assets,packages}/**/*.{js,vue,scss,less,css}' -i '**/node_modules/**','**/static/**','**/packages/kolibri-core-for-export/**'",
-    "lint-all:fix": "yarn run lint-all -w",
-    "lint:fix": "yarn run lint -w",
-    "lint:watch": "yarn lint-all -w -m",
-    "generate-locale-data": "node ./build_tools/i18n/intl_code_gen.js",
+    "fmt-frontend": "yarn run fmt-frontend:check --write",
+    "fmt-frontend:check": "kolibri-tools lint '{kolibri*/**/assets,packages}/**/*.{js,vue,scss,less,css}' --ignore '**/node_modules/**','**/static/**','**/packages/kolibri-core-for-export/**'",
+    "fmt-frontend:watch": "yarn run fmt-frontend:check --write --monitor",
+    "fmt-backend": "yarn run black-fmt --exclude '/(\\.git|\\.tox|\\.venv|build|static|dist|migrations|node_modules)/' .",
```

Comment:
Will be more efficient to pass an explicit include? Doesn't seem necessary to watch anything other than `*.py` files.

Reply:
I briefly tried explicit includes, but was concerned that over time we'd forget to keep adding them.

An aside, black's exclude logic caused me hours of grief because the docs claim you can pass multiple `--exclude` inputs but it doesn't work as expected... arg.

Reply:
If you just use a general glob pattern, this should not be much of an issue. I also think that not using it may be contributing to your infinite watch loop woes.

Reply:
unfortunately `black` only takes regex, but I can certainly do this for `chokidar`